packages/chat/tests/Unit/Response/StreamingUsageTrackerTest.php

```
@@ -0,0 +1,204 @@
+        $this->assertNotNull($result);
+        $this->assertTrue($result->isEstimated());
+        $this->assertTrue($tracker->isEstimated());
+    }
+
+    public function testRegisterCallbackAndReceiveUpdates(): void
+    {
+        $tracker = new StreamingUsageTracker();
+        $receivedUpdates = [];
+
+        $callback = new class($receivedUpdates) implements UsageCallbackInterface {
+            public function __construct(private array &$updates)
```

PHPMD (2.15.0)

Avoid unused parameters such as '$updates'. (undefined)

(UnusedFormalParameter)